### ISSUE TITLE
azy keeps leaking buffers

### ISSUE THREAD

**Issue by haolian9** [vigoux/azy.nvim#2]
from what i observed, each time i finished an azy action, two unlisted buffer will be leaked. and new action does not reuse them.
i checked line count of these buffers, they are not 0, so these buffers are not unloaded, then there are memory leaks too.

i added the following code, and the leaked buffers are gone. wish this snippet can help to address the problem.

```
   AzyUi._input_buf = vim.api.nvim_create_buf(false, true)
   vim.bo[AzyUi._input_buf].bufhidden = 'wipe'
   AzyUi._output_buf = vim.api.nvim_create_buf(false, true)
   vim.bo[AzyUi._output_buf].bufhidden = 'wipe'
```

**Comment by vigoux** [comment]
Do you mind opening a PR with this change?
Maybe using neovim api calls instead of `vim.bo`.

**Comment by haolian9** [comment]
i'd love to, but had faced some problems: i have to setup luarocks (it enforce me to install rocks in /usr/lib/luarocks/ which make me hesitated to go on), then install teal, compile to lua. that may take some time.
since this is just a little change, and i am not sure if you really want to set bufhidden to wipe, because `unload` is also good imo, especially when you want to reuse those buffers to save some time for initilizing highlight, syntax setting of buffer.
i think it's more easier to make this commit on your side :)

**Comment by vigoux** [comment]
I'll do it then.
The performance issue is not really a problem here though, all the highlighting code is actually in Lua and not set-up in advance.

By the way, the plugin ships with a `shell.nix` so if you ever use it, the development setup is as easy as `nix-shell`.

**Comment by haolian9** [comment]
haha, in the meanwhile, i found that teal provides binary releases, and made a pr

**Comment by haolian9** [comment]
should be fixed by #3
